pkg/build/nodeimage/build_impl.go

```diff
@@ -55,7 +55,7 @@ func (c *buildContext) Build() (err error) {
 func (c *buildContext) buildImage(bits kube.Bits) error {
 	// create build container
 	// NOTE: we are using docker run + docker commit so we can install
-	// debians without permanently copying them into the image.
+	// debian's without permanently copying them into the image.
```

Comment:
this is actually wrong I think, we're pluralizing "debian" as in "debian package", it's not a contraction or possessive.

Reply:
it would probably be clearer to instead say "[...] install debian packages [...]"

Reply:
https://github.com/kubernetes-sigs/kind/pull/1997/commits/bd46991731fa429cf8f5be1f2828b014080b1123